openquake/mbt/tools/fault_modeler/fault_source_modeler.py

```diff
@@ -0,0 +1,361 @@
+                                cfg_dict['config']['select_list'])
+
+        if 'param_map' in cfg_dict:
+            param_map_local.update(cfg_dict['param_map'])
+        if 'defaults' in cfg_dict:
+            defaults_local.update(cfg_dict['defaults'])
+
+    if param_map is not None:
+        param_map_local.update(param_map)
+    if defaults is not None:
+        defaults_local.update(defaults)
+
+    for key in kwargs:
+        defaults_local[key] = kwargs[key]
+
+    # Import the fault database from geojson
+    if geojson_file is not None:
+        fault_db = FaultDatabase()
+        fault_db.import_from_geojson(geojson_file,
+                                     black_list=black_list,
+                                     select_list=select_list,
+                                     param_map=param_map_local)
+    else:
+        print('Geojson file not found')
+        return
+
+    # Create the fault source model in xml_format
+    srcl = build_model_from_db(fault_db,
+                               xml_output,
+                               width_method=width_method,
+                               oqt_source=oqt_source,
+                               project_name=project_name,
+                               defaults=defaults_local)
+
+    if xml_output is None:
+        return srcl
+
+
+# -----------------------------------------------------------------------------
+
+def read_config_file(cfg_file):
+    """
+    Import various processing options from the (.ini) configuration file
+    """
+
+    cfg_dict = {}
+    cfg = configparser.RawConfigParser(dict_type=dict)
+    cfg.optionxform = str
+    cfg.read(cfg_file)
+
+    for key in ['config', 'param_map', 'defaults']:
+
+        if cfg.has_section(key):
+            cfg_dict[key] = {}
+
+            for k, v in cfg.items(key):
+                if k in option_types:
+                    cast = option_types[k]
+                else:
+                    cast = str
+                cfg_dict[key][k] = cast(v)
+
+    return cfg_dict
+
+
+# -----------------------------------------------------------------------------
+
+def build_model_from_db(fault_db,
+                        xml_output=None,
+                        width_method='seismo_depth',
+                        oqt_source=False,
+                        project_name=None,
+                        param_map=None,
+                        defaults=None,
+                        **kwargs):
+    """
+    """
+
+    param_map_local = deepcopy(fmu.param_map)
+    defaults_local = deepcopy(fmu.defaults)
+
+    if param_map is not None:
+        param_map_local.update(param_map)
+    if defaults is not None:
+        defaults_local.update(defaults)
+
+    for key in kwargs:
+        defaults_local[key] = kwargs[key]
+
+    srcl = []
+
+    for fl in fault_db.db:
+
+        try:
+            sfs_dict = fmu.construct_sfs_dict(fl,
+                                              width_method=width_method,
```

Comment:
Place a newline before ``fl`` allows a better indentation.

Reply:
All of the arguments are aligned consistently already.